packages/babel-preset-mc-app/index.js

```diff
@@ -141,7 +141,9 @@ module.exports = function getBabePresetConfigForMcApp(api, opts = {}) {
       [
         require('@babel/plugin-transform-runtime').default,
         {
-          corejs: 3,
+          // corejs messes with jest@27 in tests, due to some
+          // Promise/Date related polyfills it provides.
+          corejs: !isEnvTest && 3,
```

Comment:
Do we know if there are Node.js versions require `core-js` under test?

Alternatively we could have a `disableCoreJs` flag which however might be hard to pass from a `jest.test.config.js`. However, it could be a good idea if we want to make this change opt-in.

Reply:
Good point. Will try with node@12. Is there any commitment related lowest node version we want to support?